rackcity/api/serializers/asset_serializers.py

```diff
@@ -90,6 +95,24 @@ class Meta:
             "memory_gb",
         )
 
+class ChassisSerializer(serializers.ModelSerializer):
+    """
+    Serializers the information we want for a chassis that a blade is in (only used for serializing info to be sent)
+    """
+    rack = RackSerializer()
+
```

Comment:
adding this so that I can get the detailed rack information I need for a chassis, but avoiding having an endless loop of serializers